core/src/main/scala/org/scalarelational/column/property/NotNull.scala

```diff
@@ -1,8 +0,0 @@
-package org.scalarelational.column.property
-
-/**
- * @author Matt Hicks <matt@outr.com>
- */
-object NotNull extends ColumnProperty {
```

Comment:
The problem of removing this is you're already stepping closer to a hybrid SQL DSL instead of an "as close as possible DSL".  Part of me agrees with what you're trying to accomplish here, but it's a slippery slope leading to Slick.  Understand what I mean?  I'm not saying this isn't better, but I'm just not yet convinced.